server/config/seed.js

```diff
@@ -31,3 +31,42 @@ const insertSeed = async () => {
 };
 
 insertSeed();
+import pool from './index';
```

Comment:
Good job @chiboycalix. Kindly note that pool has already been imported on line 2. I think this second import is a mistake